src/sif/autocost.cc

```diff
@@ -45,9 +45,6 @@ constexpr float kTCUnfavorable = 2.5f;
 constexpr float kTCUnfavorableSharp = 3.5f;
 constexpr float kTCReverse = 9.5f;
 
-// How much to favor hov roads.
-constexpr float kHOVFactor = 0.85f;
```

Comment:
this is what we would want to do once we add something like a `use_hov` option, ie a way of favoring or unfavoring an HOV edge.